RUN3/AliAnalysisTaskAO2Dconverter.cxx

```diff
@@ -644,26 +645,53 @@ void AliAnalysisTaskAO2Dconverter::InitTF(ULong64_t tfId)
     tCaloTrigger->SetBasketSize("*", fBasketSizeEvents);
   }
 
-  // Associuate branches for MUON tracks
-  TTree *tMuon = CreateTree(kMuon);
-  if (fTreeStatus[kMuon])
+  // Associate FwdTrack branches for MUON tracks
+  TTree *tFwdTrack = CreateTree(kFwdTrack);
+  if (fTreeStatus[kFwdTrack])
   {
```

Comment:
@vfeuilla, @jgrosseo 
This will indeed be needed. For the converted data, at the moment, the fIndexCollisions is assigned for events with a vertex. If there are events with muon tracks, but no vertex, what should be the behaviour? Would it be fine to have a default index, like -1 ?